.github/workflows/check-outdated-trusted-root.yml

```diff
@@ -0,0 +1,24 @@
+name: "Check for outdated trusted root certificates"
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Pull new trusted root
+        run: gh attestation trusted-root > resources/new-trusted-root.jsonl
+      - name: Create an issue if different
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          diff resources/trusted-root.jsonl resources/new-trusted-root.jsonl \
+            && echo "Trusted root cert has not changed, no action required." \
+            || ( \
+              (gh issue list | grep -i "Trusted root needs updating") \
```

Comment:
I think this might not work if too many open issues exist. This should not happen in practice, but perhaps you could make this a little smarter by leveraging the filter flags: https://cli.github.com/manual/gh_issue_list

Something: `gh issue list --author "github-actions[bot]"` might help. Or just create a dedicated label.